doc/api/worker_threads.md

```diff
@@ -194,6 +194,38 @@ isMarkedAsUntransferable(pooledBuffer);  // Returns true.
 
 There is no equivalent to this API in browsers.
 
+## `worker.markAsUncloneable(object)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `object` {any} Any arbitrary JavaScript value.
+
+Mark an object as not cloneable. If `object` is used as [`message`](#event-message) in
+a [`port.postMessage()`][] call, an error is thrown. This is a no-op if `object` is a
+primitive value.
+
+Note, this does not take effect on `ArrayBuffer`, or any `Buffer` like objects.
```

Comment:
My understanding is `Buffer` like objects are from v8 and they have their own `SerializeDelegate` to handle the Serialization, so marking node's internal flag on or off will not have control over there.